libs/community/langchain_community/document_loaders/recursive_url_loader.py

```diff
@@ -313,6 +314,16 @@ def simple_metadata_extractor(
                 encoding, unless the `encoding` argument has already been explicitly set.
             encoding: The encoding of the response. If manually set, the encoding will be
                 set to given value, regardless of the `autoset_encoding` argument.
+            proxies: A dictionary mapping protocol names to the proxy URLs to be used for requests.
```

Comment:
Looks good, any chance you'd be willing to add a sentence to the security note so that folks know they can specify proxies?